microsoft-edge/webdriver-chromium.md

```diff
@@ -183,6 +214,19 @@ service_args = ['--verbose']
 driver = Edge(service_args = service_args)
 ```  
 
+#### [JavaScript](#tab/javascript/)  
+
+<a id="customizing-microsoft-edge-driver-services-code" />  
+
+When using JavaScript, the `edge` object has a `ServiceBuilder` method that returns a `service` you may optionally pass to the `edge.driver` object, which creates and starts the service for you. 
```

Comment:
`ServiceBuilder` is a class, not a method, and the choice of `edge` as a namespace is arbitrary and up to the user, not us (i.e. the user happens to name us `edge` when they require() us). Instead, can we say something like?

Use the `ServiceBuilder` class to create and configure a `Service`. You may optionally pass this `Service` object to the `Driver` object which will start and stop the service for you.